app/controllers/v1/facilities/ccp_controller.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'will_paginate/array'
+class V1::Facilities::CcpController < FacilitiesController
+  # Provider supports the following query parameters:
+  # @param bbox - Bounding box in form "xmin,ymin,xmax,ymax" in Lat/Long coordinates
+  # @param services - Optional specialty services filter
+  def index
+    api_results = ppms_search.collect do |result|
+      PPMS::Provider.new(
+        result.attributes.transform_keys { |k| k.to_s.snakecase.to_sym }
+      )
+    end.uniq(&:id).paginate(pagination_params)
+
+    render_json(serializer, ppms_params, api_results, { include: [:specialties] })
```

Comment:
Why not just:
```ruby
render_json(PPMS::ProviderSerializer, ppms_params, api_results, { include: [:specialties] })
```
and get rid of `def serializer`?

Reply:
ok